src/main/java/Duke.java

```diff
@@ -1,10 +1,23 @@
 public class Duke {
+
+    private static void showDivider() {
+        System.out.println("-----------------------------------------------------");
```

Comment:
I like the way that you declare a constant here to make the code clean! And good job remembering the format. Most of us would just forget it.